### ISSUE TITLE
CAN bus bundle only supports 11-bit identifiers

### ISSUE THREAD

**Issue by dwoodard1** [eclipse/kura#555]
Currently, the CAN bus bundle only supports 11-bit identifiers (SFF). This should be extended to include 29-bit identifiers (EFF). The code to be updated is here:

``` java
CanMessage cm = new CanMessage();
cm.setCanId(ci.getCanId_SFF());
cm.setData(cf.getData());
return cm;
```

**Comment by MMaiero** [comment]
@pierrepitiot Is this issue addressed by the latest changes you made?

**Comment by pierrepitiot** [comment]
@MMaiero yes, this has been fixed by PR #994
